src/daos/docker/update_confluence.py

```diff
@@ -0,0 +1,158 @@
+from bs4 import BeautifulSoup   # pip3 install beautifulsoup4
+import json
+import requests # pip3 install requests
+import base64
+from requests.auth import HTTPBasicAuth
+import copy
+import csv
+import sys
+import os
+
+# https://developer.atlassian.com/cloud/confluence/rest/api-group-content/#api-wiki-rest-api-content-id-put
+auth_email = os.environ['CONFLUENCE_UPDATE_EMAIL']
+auth_key = os.environ['CONFLUENCE_UPDATE_KEY']
+auth_str = auth_email + ":" + auth_key
+auth_bytes = auth_str.encode('ascii')
+auth_encoded = base64.b64encode(auth_bytes)
+auth_basic = auth_encoded.decode('ascii')
+auth_token = "Basic " + auth_basic
+
+def create_payload(json_page):
+   payload_json = {
+   "version": {
+      "number": 5
+   },
+   "title": "<string>",
+   "type": "<string>",
+   "status": "current",
+   # "ancestors":[{"id":"458822","key":"PUB"}],
+   # "ancestors": [
+   #    {
+   #       "id": 2154
+   #    }
+   # ],
+    "space": {
+        "id": 458755,
+        "key": "PUB",
+    },
+   "body": {
+      #  "view": {
+      #    "value": "<string>",
+      #    "representation": "view"
+      #  },
```

Comment:
What's up with the commented out code here?

Reply:
my bad, I guess I forgot to remove them